Python/Product/PythonTools/PythonTools/LanguageServerClient/PythonLanguageClient.cs

```diff
@@ -242,7 +242,7 @@ private async Task OnWorkspaceFolders(object sender, WorkspaceFolders.WorkspaceF
         public async Task OnServerInitializedAsync() {
             IsInitialized = true;
             // Set _workspaceFoldersSupported to true and send to either workspace open or solution open
-            OnWorkspaceFolderWatched(this, EventArgs.Empty);
+            await OnWorkspaceOrSolutionOpened(this, EventArgs.Empty);
```

Comment:
this is the key to the fix.. use and async function and await.. instead of sync func that will trigger async later